### ISSUE TITLE
可微分计算图的实现 - 邢存远的博客 | Welt Xing's Blog

### ISSUE THREAD

**Issue by Kaslanarian** [Kaslanarian/Kaslanarian.github.io#163]
https://welts.xyz/2022/04/18/naive_graph/ 

 老访客请刷新获取最新内容

**Comment by YerongLi** [comment]
https://github.com/Kaslanarian/PyAdNet/blob/main/naive_example.py这个连接已经不存在了，请问有最新的代码吗？

**Comment by Kaslanarian** [comment]
> https://github.com/Kaslanarian/PyAdNet/blob/main/naive_example.py 这个连接已经不存在了，请问有最新的代码吗？

我已经把这部分代码作为PyDyNet的一部分了，随着更新被迭代得差不多了，你可以参考下我的PyDyNet库

**Comment by YerongLi** [comment]
I don't think it is in PyAdNet, it might be in another repo. PyAdNet did not do manual forward/backward with topology sort.
